tests/deployment_utils.py

```diff
@@ -173,7 +174,7 @@ def __init__(self, perforce_workspace, git_client, local_sources):
     def set_environment(self, execution_environment):
         self.environment = execution_environment
         self.working_dir = self.environment.get_working_directory()
-        self.project_root = os.path.join(self.working_dir, "temp")
+        self.project_root = self.working_dir if self.nonci else os.path.join(self.working_dir, "temp")
```

Comment:
Do we need a one-line if-statement here?

Reply:
yes